troupon/static/scss/intlTelInput.scss

```diff
@@ -0,0 +1,281 @@
+      background-color: transparent;
+    }
+  }
+
+  .selected-flag {
+    // render above the input
+    z-index: 1;
+    position: relative;
+    width: $selectedFlagWidth;
+    // this must be full-height both for the hover highlight, and to push down the
+    // dropdown so it appears below the input
+    height: 100%;
+    padding: 0 0 0 $flagPadding;
+
+    // vertically center the flag
+    .iti-flag {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      margin: auto;
+    }
+
+    .iti-arrow {
+      position: absolute;
+      // split the difference between the flag and the arrow height to verically center
+      top: 50%;
+      margin-top: -1 * ($arrowHeight / 2);
+      right: $arrowPadding;
+
+      // css triangle
+      width: 0;
+      height: 0;
+      border-left: $triangleBorder solid transparent;
+      border-right: $triangleBorder solid transparent;
+      border-top: $arrowHeight solid $arrowColor;
+
+      &.up {
+        border-top: none;
+        border-bottom: $arrowHeight solid $arrowColor;
+      }
+    }
+  }
+
+  // the dropdown
+  .country-list {
+    // override default list styles
+    list-style: none;
+    position: absolute;
+    // popup so render above everything else
+    z-index: 2;
+
+    // place menu above the input element
+    &.dropup {
+      bottom: 100%;
+      margin-bottom: (-$borderWidth);
+    }
+
+    // dropdown flags need consistent width, so wrap in a container
+    .flag-box {
+      display: inline-block;
+      width: $flagWidth;
+    }
+
+    padding: 0;
+    // margin-left to compensate for the padding on the parent
+    margin: 0 0 0 (-$borderWidth);
+
+    box-shadow: 1px 1px 4px rgba(0,0,0,0.2);
+    background-color: white;
+    border: $borderWidth solid $greyBorder;
+
+    // don't let the contents wrap AKA the container will be as wide as the contents
+    white-space: nowrap;
+    // except on small screens, where we force the dropdown width to match the input
+    @media (max-width: 500px) {
+      white-space: normal;
+    }
+
+    max-height: 200px;
+    overflow-y: scroll;
+
+    // the divider below the preferred countries
+    .divider {
+      padding-bottom: 5px;
+      margin-bottom: 5px;
+      border-bottom: $borderWidth solid $greyBorder;
+    }
+
+    // each country item in dropdown (we must have separate class to differentiate from dividers)
+    .country {
+      // Note: decided not to use line-height here for alignment because it causes issues e.g. large font-sizes will overlap, and also looks bad if one country overflows onto 2 lines
+      padding: 5px 10px;
+      // the dial codes after the country names are greyed out
+      .dial-code {
+        color: $greyText;
+      }
+    }
+    .country.highlight {
+      background-color: $hoverColor;
+    }
+
+    // spacing between country flag, name and dial code
+    .flag-box, .country-name, .dial-code {
+      vertical-align: middle;
+    }
+    .flag-box, .country-name {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line